stdlib/public/core/Runtime.swift.gyb

```diff
@@ -408,37 +408,17 @@ internal func _float${bits}ToString(
   _ value: Float${bits}, debug: Bool
 ) -> String {
 
-  if !value.isFinite {
-    let significand = value.significandBitPattern
-    if significand == 0 {
-      // Infinity
+  if value.isInfinite {
```

Comment:
This could be omitted if `inf` is handled at the next layer out.

Reply:
I've removed infinity handling here.  Infinity is now handled in two places:  In the outermost Swift code for `description` and `debugDescription` (which return constant strings) and in the actual C formatting code.  (Because of the former, the latter is not technically necessary except to ensure that the C formatter is actually complete.)